### PR TITLE
py-yahmm: new package at 1.1.3

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-yahmm/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    homepage = "http://pypi.python.org/pypi/yahmm/"
+    url      = "https://pypi.io/packages/source/y/yahmm/yahmm-1.1.3.zip"
+
+    version('1.1.3', sha256='fe3614ef96da9410468976756fb93dc8235485242c05df01d8e5ed356a7dfb43')
+
+    depends_on('py-cython', type=('build', 'run'))
+    depends_on('py-matplotlib@1.3.1:', type=('build', 'run'))
+    depends_on('py-networkx@1.8.1:', type=('build', 'run'))
+    depends_on('py-nose@1.3.3:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-yahmm/package.py:20]: Missing dependencies on:
```python
        "numpy >= 1.8.0",                                                       
        "scipy >= 0.13.3",      
```

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-yahmm/package.py]: updated!

[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-yahmm/package.py]: updated!

File: var/spack/repos/builtin/packages/py-yahmm/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class PyYahmm(PythonPackage):
+    """YAHMM is a HMM package for Python, implemented in Cython for speed."""
+
+    homepage = "http://pypi.python.org/pypi/yahmm/"
+    url      = "https://pypi.io/packages/source/y/yahmm/yahmm-1.1.3.zip"
+
+    version('1.1.3', sha256='fe3614ef96da9410468976756fb93dc8235485242c05df01d8e5ed356a7dfb43')
+
+    depends_on('py-cython', type=('build', 'run'))
```
[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-yahmm/package.py]: updated!

**Pull request merged**